src/box/lua/net_box.lua

```diff
@@ -59,11 +60,23 @@ local IPROTO_FEATURE_NAMES = {
     [4]     = 'pagination',
 }
 
+local check_select_features = function(opts, remote)
```

Comment:
Nit: Missing comment. Passing feature set instead of `remote` looks more appropriate. Anyway, the function is used only once, we're not planning to use anywhere else, and it's just a few lines long so I'd inline it.

Reply:
Done